### DISCUSSION TITLE
Routing Jack on Windows Guide Fix for ASIO4ALL

### DISCUSSION THREAD

**Discussion by chairbender** [jamulussoftware/jamulus#2554] [Documentation]
EDIT: Sigh, nevermind...it looks like I am still having issues again even after doing this. I swear it fixed my problems on one day, but I guess it isn't consistent...

Hi, I found this page (How to play audio from your PC to Jamulus (using JACK)) very useful for trying to set things up on Windows 10 using ASIO4ALL: https://jamulus.io/kb/2022/01/26/Routing-JACK-Windows.html

However, I ran into a frustrating issue where, when I try to press the  "start" button in QjackCtl with ASIO4ALL as the interface, it (usually) fails to startup and shows an error. After much trial and error, I discovered what fixes it is to go into advanced and change the "Start Delay" to 10 seconds. The default of 2 seems to be insufficient time. Perhaps it is because ASIO4ALL takes longer to start up, as it needs to start up when QjackCtl starts as opposed to other ASIO devices which are constantly running?

Anyway, to save others the trouble, if possible, I would suggest updating the guide I linked to update this section like so:

> Ensure the Sample Rate is set to 48000, the driver is portaudio, Frames/Period is set to something low like 64 or 128. **Go to the "Advanced" tab and ensure "Start Delay" is set to 10 seconds**. You can now close this window with “OK”. Next, click on the Start button in Qjackctl and open Jamulus and go into settings. There select “JackRouter” as device:

**Comment by ann0see** [comment]
Hi @chairbender great you find my guide useful. As I noted in it, it’s not perfect – and probably improbable. The addition you mentioned is a quick fix. 
Feel free to edit it yourself (which should suggest you to open a pull request): https://github.com/jamulussoftware/jamuluswebsite/edit/release/_posts/2022-01-26-Routing-JACK-Windows.md

Concerning the other problems: what are they? Can you use different ASIO drivers?
